lib/api/core/models/repositories/userRepository.js

```diff
@@ -9,130 +11,134 @@ var
 /**
  * @class UserRepository
  * @extends Repository
- * @param {Kuzzle} kuzzle
- * @param opts
- * @constructor
  */
-function UserRepository (kuzzle, opts) {
-  Repository.apply(this, arguments);
-  this.collection = 'users';
-  this.ObjectConstructor = User;
-
-  if (opts !== undefined && opts.ttl !== undefined) {
-    this.ttl = opts.ttl;
+class UserRepository extends Repository {
+  /**
+   * @param {Kuzzle} kuzzle
+   * @param opts
+   * @constructor
+   */
+  constructor (kuzzle, opts) {
```

Comment:
@benoitvidis said me the RoleRepository and ProfileRepository was ment to force the disabling of Redis cache as they already have inmemory cache, but was somehow re-enabled by mistake, I'll create an issue about it to re-disable Redis cache.